include/ibm/locks.hpp

```diff
@@ -124,8 +109,6 @@ class Lock
 
     Lock()
     {
-        // Remove the persistent file
-        std::filesystem::remove(fileName);
         lockTable.clear();
         transactionId = lockTable.empty() ? 0 : prev(lockTable.end())->first;
```

Comment:
this is where I am confused, if we remove the persistency the transaction id's will be starting from 0 every time bmcweb restarts or when BMC takes a reboot, so there is a possibility for HMC to see the same transaction ID for different locks across reboot. Are they okay with that ?

Reply:
They had mentioned that they will work good when they re-acquire the locks. I have asked them again to re-confirm. Lets wait for their answer and i will update here

Reply:
Seems like this concern is OK for HMC team.

Reply:
sounds good